packages/eslint-config-react-app/index.js

```diff
@@ -241,15 +224,15 @@ module.exports = {
     ],
 
     // https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules
-    'import/first': 'error',
-    'import/no-amd': 'error',
-    'import/no-webpack-loader-syntax': 'error',
+    'import/first': 'warn',
```

Comment:
This should stay an error. People often misunderstand that ES imports are hoisted, and try to do side effects between them. It's very confusing that it doesn't work. This lint rule altogether prevents this from happening and trains you to write imports in the way the bundler actually interprets them.